daffodil-test/src/test/scala/org/apache/daffodil/section05/simple_types/TestSimpleTypes2.scala

```diff
@@ -79,6 +81,11 @@ class TestSimpleTypes2 {
   @Test def test_bitOrderChangeInvalid2() { runner1.runOneTest("bitOrderChangeInvalid2") }
   @Test def test_bitOrderChangeInvalid2Unparser() { runner1.runOneTest("bitOrderChangeInvalid2Unparser") }
   @Test def test_noByteOrder() { runner1.runOneTest("noByteOrder") }
+  //@Test def test_bitOrderChange() { runner1.runOneTest("bitOrderChange") }
```

Comment:
Same issue this file. No JIRA ticket means these tests won't get revisited even if a related bug is fixed. Add "// NO JIRA TICKET" so we can revisit and label these (unless you want to label them now, but someone else can do this.)